core/io/dir_access.cpp

```diff
@@ -323,6 +324,33 @@ Ref<DirAccess> DirAccess::create(AccessType p_access) {
 	return da;
 }
 
+Ref<DirAccess> DirAccess::create_tmp(const String &p_prefix, bool p_keep) {
+	Ref<DirAccess> dir_access = DirAccess::open(OS::get_singleton()->get_tmp_path());
+	String hash;
```

Comment:
Out of curiosity, wouldn't using the timestamp be sufficient? Is there an additional role the `hash` is playing?

Reply:
To be honest, I didn't think about the timestamp at all. Could be sufficient indeed.

And no, there's no additional role to the hash other than just making a guess on a not used file name.

Reply:
I converted `hash` to a datetime string + usec ticks.